MeetU/MeetU/Content/profile.less

```diff
@@ -2,240 +2,385 @@
-                        border-bottom:1px solid @splitter-color-light;
-                        padding-top:6px;
-                        /*height:74px;*/
-                        .meetup-title{
-                            color:@color-coral;
-                            /*font-weight:bold;*/
-                        }
+
+                .profile-content-right-container {
+                    padding: 0px 0px 0px 15px;
+
+                    @media only screen and (max-width:767px) {
+                        padding: 0px;
                     }
 
-                    .hosted-meetup:last-child{
-                        border-bottom:none;
+                    .content-main-body {
+                        .personal-information {
+                        }
+
+                        .joined-list {
+                        }
+
+                        .launched-list {
+                        }
+
+                        .content-main-title {
+                            border-bottom: 1px solid @splitter-color-dark;
+                            padding: 2px 16px 0px 16px;
+                            color: @text-color-asherstone;
+                            font-size: @meetup-font-size-sm;
+
+                            @media only screen and (max-width:767px) {
+                                display: none;
+                            }
+                        }
                     }
                 }
             }
         }
 
-    }
-    @media only screen and (max-width: 991px){
-        .profile-main-container{
-            width:750px;
-        }
-    }
+        .profile-side-container {
+            .profile-side-title {
+                padding: 2px 16px 0px 16px;
+                border-bottom: 1px solid @splitter-color-dark;
+                color: @text-color-asherstone;
+                font-size: @meetup-font-size-sm;
+            }
 
-    @media only screen and (max-width: 767px){
-        .profile-main-container{
-            width:100%;
+            .side-section {
+                min-height: 120px;
+                text-align: center;
+                padding: 10px 20px;
+                color: @text-color-asherstone;
+                font-size: @meetup-font-size-xs;
+            }
+
+            .side-section:last-child {
+                border-bottom: none;
+            }
         }
     }
-    
 }
 
-.block{
+.block-xs {
+}
 
+.color-male {
+    color: #33ccff;
+    font-weight: bold;
 }
 
-@media only screen and (max-width: 767px){
-    .block{
-        display:none;
-    }
+.color-female {
+    color: #ffb3b3;
+    font-weight: bold;
 }
 
-.section{
-    border-bottom:1px solid @splitter-color-medium;
-    min-height:200px;
-    padding: 6px 16px;
-    .section-header{
-        padding-left:14px;
-        padding-right:6px;
-        padding-top:4px;
-        padding-bottom:4px;
-        font-weight:bold;
-        font-size:@meetup-font-size-md;
-        .header-more{
-            float:right;
+.meetup-wrapper {
```

Comment:
can you make sure this class meetup-wrapper not being used somewhere else? actually you should always put profile- prefix in it, eg. profile-meetup-wrapper. I found some similar problems in this file.